### PR TITLE
Regen bigquerydatatransfer with toolkit changes (WIP - do not merge)

### PR DESCRIPTION
**Pull request by andreamlin**
Using gapic-generator at https://github.com/andreamlin/toolkit/tree/python_local_shared_imports.

### CODE REVIEW

File: bigquery_datatransfer/nox.py [hunk 1]
```diff
@@ -53,7 +53,7 @@ def default(session):
 
 
 @nox.session
-@nox.parametrize('py', ['2.7', '3.5', '3.6', '3.7'])
+@nox.parametrize('py', ['2.7', '3.5', '3.6'])
```
Comment (tseaver) [inline bigquery_datatransfer/nox.py]: We should be leaving the `3.7` in the list.

File: bigquery_datatransfer/google/cloud/bigquery_datatransfer_v1/proto/datatransfer_pb2_grpc.py [hunk 2]
```diff
@@ -1,9 +1,9 @@
 # Generated by the gRPC Python protocol compiler plugin. DO NOT EDIT!
 import grpc
 
-import google.cloud.bigquery_datatransfer_v1.proto.datatransfer_pb2 as google_dot_cloud_dot_bigquery_dot_datatransfer__v1_dot_proto_dot_datatransfer__pb2
-import google.cloud.bigquery_datatransfer_v1.proto.transfer_pb2 as google_dot_cloud_dot_bigquery_dot_datatransfer__v1_dot_proto_dot_transfer__pb2
-import google.protobuf.empty_pb2 as google_dot_protobuf_dot_empty__pb2
+from google.cloud.bigquery.datatransfer_v1.proto import datatransfer_pb2 as google_dot_cloud_dot_bigquery_dot_datatransfer__v1_dot_proto_dot_datatransfer__pb2
+from google.cloud.bigquery.datatransfer_v1.proto import transfer_pb2 as google_dot_cloud_dot_bigquery_dot_datatransfer__v1_dot_proto_dot_transfer__pb2
```
Comment (tseaver) [inline bigquery_datatransfer/google/cloud/bigquery_datatransfer_v1/proto/datatransfer_pb2_grpc.py]: The underscore is crucial, here.

File: bigquery_datatransfer/google/cloud/bigquery_datatransfer_v1/proto/datatransfer_pb2.py [hunk 3]
```diff
@@ -14,7 +14,7 @@
 
 
 from google.api import annotations_pb2 as google_dot_api_dot_annotations__pb2
-from google.cloud.bigquery_datatransfer_v1.proto import transfer_pb2 as google_dot_cloud_dot_bigquery_dot_datatransfer__v1_dot_proto_dot_transfer__pb2
+from google.cloud.bigquery.datatransfer_v1.proto import transfer_pb2 as google_dot_cloud_dot_bigquery_dot_datatransfer__v1_dot_proto_dot_transfer__pb2
```
Comment (tseaver) [inline bigquery_datatransfer/google/cloud/bigquery_datatransfer_v1/proto/datatransfer_pb2.py]: The underscore is crucial, here.

File: bigquery_datatransfer/google/cloud/bigquery_datatransfer_v1/proto/datatransfer_pb2_grpc.py (hunk 2 above)
Comment (andreamlin) [reply bigquery_datatransfer/google/cloud/bigquery_datatransfer_v1/proto/datatransfer_pb2_grpc.py]: Definitely noted.

File: bigquery_datatransfer/google/cloud/bigquery_datatransfer_v1/proto/datatransfer_pb2.py (hunk 3 above)
Comment (andreamlin) [reply bigquery_datatransfer/google/cloud/bigquery_datatransfer_v1/proto/datatransfer_pb2.py]: @lukesneeringer @landrito @eoogbe 
Is this file a protoc artifact? Do we have any influence over the output of whatever tool makes this file?

**Pull request closed without merging**